script/cadence/transactions/create-proxy-account.cdc

```diff
@@ -2,14 +2,11 @@ import FlowColdStorageProxy from 0xProxy
 import FlowManager from 0xFlow
```

Comment:
I don't see `FlowManager` in this PR or anywhere on Testnet or Mainnet - is this contract & transaction used purely for testing?

Reply:
Yeah I'm wondering if this is to work around to change state in the testing environment that should actually just be exposed as a method in the test framework. Come to think of it, it probably is since the newly created account (created in `.setup`) otherwise wouldn't be accessible in the test environment - I've run into that when using the Cadence test framework

Reply:
Yeah, it's probably something like that. Only 3 of the transactions in this folder are actually used live in the running system, so this is probably some unfinished work